.gitignore

```diff
@@ -0,0 +1,5 @@
+.vscode
```

Comment:
Do czego jest potrzebny ten plik?

Reply:
Który pliki? .gitignore? Wymusza aby pliki wskazane wzrocem nie były brane pod uwagę przy pracy repozytorium. 
Pliki w .vscode to ustawienia projektu generowane przez vscode